supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,29 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[commands-with-elevated-privileges]]
+== Commands with elevated privileges
+
+Some commands require elevated privileges to run. Users without elevated privileges must first do one of the following to run such a command:
+
+* Use `su -i` to switch to the root user account.
+* Preface the command with `sudo` to temporarily change their current privileges.
+
+=== Guidelines for documenting commands with elevated privileges
+
+When documenting commands that require elevated privileges, consider the following guidelines and formulate a consistent approach to use throughout your content:
+
+* If you include a shell prompt in a sample command, always show the correct prompt for a regular user (`$`) or a user with elevated privileges (`#`).
++
+[NOTE]
+====
+Do not rely solely on a shell prompt in a sample command to indicate the required privilege level to run a command.
+====
++
+* If a command requires temporarily elevated privileges, use `sudo` at the beginning of the sample command syntax.
+* If multiple commands in a procedure require elevated privileges, consider adding introductory content to tell the user what to do. For example, "The commands in this procedure require elevated privileges to run. Either switch to the root user using `su -i` or preface the commands with `sudo`."
```

Comment:
- A similar suggestion to the previous comment to remove the 'consider'
- And suggesting to change last sentence to rework the potentially ambiguous usage of 'using' (see the ISG for info about 'using').

`* If multiple commands in a procedure require elevated privileges, add introductory content to tell the user what to do. For example, "The commands in this procedure require elevated privileges to run. Either use  `su -i`  to switch to the root user or preface the commands with `sudo`."`

Reply:
Thanks. Great edit!